packages/grid/src/GridRendererTypes.ts

```diff
@@ -0,0 +1,65 @@
+import GridMetrics, { VisibleIndex, Coordinate } from './GridMetrics';
+import GridModel from './GridModel';
+import GridRange from './GridRange';
+import { GridTheme } from './GridTheme';
+import { GridSeparator } from './mouse-handlers';
```

Comment:
Import directly from `GridSeparatorMouseHandler`:
```suggestion
import { GridSeparator } from './mouse-handlers/GridSeparatorMouseHandler';
```